src/sched/lthread.c

```diff
@@ -403,7 +394,7 @@ void _lthread_free(struct lthread* lt)
         lthread_rundestructors(lt);
     if (lt->itls != 0)
     {
-        munmap(lt->itls, lt->itlssz);
+        enclave_munmap(lt->itls, lt->itlssz);
```

Comment:
These look like upstream changes too. Can you rebase?

Reply:
yup